kotlintest-tests/kotlintest-tests-core/src/test/kotlin/com/sksamuel/kotlintest/matchers/throwable/ThrowableMatchersTest.kt

```diff
@@ -0,0 +1,178 @@
+package com.sksamuel.kotlintest.matchers.throwable
+
+import io.kotlintest.matchers.throwable.*
+import io.kotlintest.shouldThrow
+import io.kotlintest.shouldThrowAny
+import io.kotlintest.shouldThrowExactly
+import io.kotlintest.specs.FreeSpec
+import java.io.FileNotFoundException
+import java.io.IOException
+
+class ThrowableMatchersTest : FreeSpec() {
+  init {
+    "shouldThrowAny" - {
+      "shouldBeExactlyOfType" - {
```

Comment:
If I remember correctly, the final leaf of a FreeSpec must not contain a `-`.

Would you mind changing this? If you rather use another style, there's no problem

Reply:
done!